### ISSUE TITLE
Browse and subset tab: Show numeric/date variable with only one value as a data frequency table

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB/web-eda#938]
Danielle made changes to the use of datashape, so now numeric and date variables that have <10 values are shown using a histogram in the browse and subset tab. This works well until you get down to very few values that are clustered together. In particular, if there is a single value, the histogram shows a bar stretching the whole width of the plot. In these cases where there is just one value, we'd like the variable to show as data frequency table. 

An alternative solution is for the ClinEpi outreach team to manually annotate "forceStringType" for these variables, but that requires manual curation, which is more error prone, so we'd like to avoid that.

**Comment by chowington** [comment]
Hi @danicahelb, I know it's been a while since this ticket was made, but do you know of a variable that fits this description?

**Comment by chowington** [comment]
And by "data frequency table", do you mean the view with horizontal red bars that's shown for categorical variables?

**Comment by chowington** [comment]
I'll tag @SheenaTomko for the above questions also

**Comment by SheenaTomko** [comment]
Yes, for "data frequency table", I mean the view that's shown for categorical variables.

As for examples, that's hard since I updated the min/max range for the variables that I found with that issue. Here are 2 that used to show up as a fat bar that I adjusted the range on. If needed, I could ask Jay to reload this study and remove the min/max annotation for these variables for you to test with
https://qa.restricted.clinepidb.org/ce.restricted.qa/app/workspace/analyses/DS_7cea54058d/new/variables/EUPATH_0000738/EUPATH_0041040
https://qa.restricted.clinepidb.org/ce.restricted.qa/app/workspace/analyses/DS_7cea54058d/new/variables/EUPATH_0000738/EUPATH_0041050

**Comment by chowington** [comment]
Hmm, yes that would be helpful if it's possible and not too complicated. The other options are to either (2) find another such variable, or (3) consider whether it's worth fixing this issue now, because if there are no cases in the foreseeable future where the issue happens, it may not be worth addressing now.

**Comment by danicahelb** [comment]
@chowington, I'm moving this ticket to Backlog, it is not worth addressing in b62. We will take a screenshot if we see this issue again in the future

**Comment by SheenaTomko** [comment]
For future reference when working on this ticket, the variables "Hematological, lymphatic condition end date" EUPATH_0041050 and "Endocrine condition start date" EUPATH_0041040 in Kalifabougou malaria cohort should have this issue again soon. It will first be in the Kalifabougou malaria qa dataset later this week and later in the main Kalifabougou malaria cohort datasets when we release the qa dataset.
